config/GregTech/Recipes.cfg

```diff
@@ -140857,8 +140857,8 @@ pulveriser {
     I:blockChromiumDioxide_756=756
     I:blockChromo-Alumino-Povondraite_400=400
     I:blockChromo-Alumino-Povondraite_882=882
-    I:blockClay_135=135
-    I:blockClay_400=400
+    I:blockClay_135=-135
```

Comment:
I:blockClay_135=0

Reply:
> set it to any nonpositive value will disable the recipe

Is what glease said. 0 is positive, so I dunno if it works, but I'll try it

Reply:
i did all recipes i dont want with 0 and it works

Reply:
It works, so I changed them. Personally, I think - is better since it's possible to search for, but whatever.